unit1-wrap-up/activity-mind-map.md

```diff
@@ -0,0 +1,13 @@
+# Unit 1 - Reflection Activity
+
+Wow! We've learned so much in the last 5 weeks. Let's take time to reflect on our learning and make connections.
```

Comment:
Because it'd be great to not tie our curriculum to time passing, could we divorce it from "five weeks"?

```suggestion
Wow! We've learned so much in this unit. Let's take time to reflect on our learning and make connections.
```